app/components/op_primer/border_box_table_component.html.erb

```diff
@@ -34,13 +34,18 @@ See COPYRIGHT and LICENSE files for more details.
       test_selector:,
     )) do |component|
     component.with_header(classes: grid_class, color: :muted) do
+      concat render(Primer::Beta::Text.new(classes: "op-border-box-grid--mobile-heading",
+                                           font_weight: :semibold)) { mobile_title }
       headers.each do |name, args|
         caption = args.delete(:caption)
-        concat render(Primer::Beta::Text.new(font_weight: :semibold, **header_args(name))) { caption }
+        concat render(Primer::Beta::Text.new(classes: header_classes(name),
+                                             font_weight: :semibold,
+                                             **header_args(name))) { caption }
       end
 
       if has_actions?
-        concat render(Primer::BaseComponent.new(tag: :div))
+        concat render(Primer::BaseComponent.new(classes: "op-border-box-grid--heading",
```

Comment:
`op-border-box-grid--heading` should probably be a variable as it is reused in `header_classes`.